vendor/vendor.json

```diff
@@ -309,6 +309,14 @@
 			"versionExact": "v1.10.25"
 		},
 		{
+			"checksumSHA1": "hxexHJf5LWfELTK4wImDFlBJYfc=",
+			"path": "github.com/aws/aws-sdk-go/service/batch",
+			"revision": "47ba3b0518c83189f3a1c95d20fc22f267e02ed3",
+			"revisionTime": "2017-06-13T17:42:44Z",
+			"version": "v1.8.41",
+			"versionExact": "v1.8.41"
```

Comment:
As we are using the `v1.10.25` version from the AWS Go SDK, could you run `govendor fetch github.com/aws/aws-sdk-go/service/batch@=v1.10.25` ?

Thanks!